src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -447,14 +406,14 @@ private Object getInstance(final String className) throws GameParseException {
       instance = instanceClass.getDeclaredConstructor().newInstance();
       // a lot can go wrong, the following list is just a subset of potential pitfalls
     } catch (final ClassNotFoundException e) {
-      throw new GameParseException(mapName, "Class <" + className + "> could not be found.");
+      throw new GameParseException(mapName, "Class <" + className + "> could not be found", e);
     } catch (final InstantiationException e) {
       throw new GameParseException(mapName,
-          "Class <" + className + "> could not be instantiated. ->" + e.getMessage());
+          "Class <" + className + "> could not be instantiated", e);
     } catch (final IllegalAccessException e) {
-      throw new GameParseException(mapName, "Constructor could not be accessed ->" + e.getMessage());
+      throw new GameParseException(mapName, "Constructor could not be accessed", e);
     } catch (ReflectiveOperationException e) {
-      throw new GameParseException(mapName, "Exception while constructing object ->" + e.getMessage());
+      throw new GameParseException(mapName, "Exception while constructing object", e);
```

Comment:
Now that you are including the cause in `GameParseException`, I don't think it's necessary to have different messages for different types of exceptions.  You should be able to reduce this entire method to the following:

```java
private Object getInstance(final String className) throws GameParseException {
  try {
    return Class.forName(className).getDeclaredConstructor().newInstance();
  } catch (final ReflectiveOperationException e) {
    throw new GameParseException(mapName, String.format("Unable to create instance of class <%s>", className), e);
  }
}
```